lib/workos/user_management.rb

```diff
@@ -481,19 +489,30 @@ def authenticate_user_with_code(code:, client_id:, ip_address: nil, user_agent:
           client_id: String,
           pending_authentication_token: String,
           authentication_challenge_id: String,
+          ip_address: T.nilable(String),
+          user_agent: T.nilable(String),
         ).returns(WorkOS::UserResponse)
       end
-      def authenticate_user_with_totp(code:, client_id:, pending_authentication_token:, authentication_challenge_id:)
+      def authenticate_with_totp(
```

Comment:
I believe this is still in ongoing development? CC @amygdalama

Reply:
yeah i assume we are shipping with this but it was missing from the docs. i believe dane was gonna ask hadley about it today.